admin/hotlist-service/src/main/java/io/mosip/hotlist/event/HotlistEventHandler.java

```diff
@@ -50,7 +49,7 @@ public class HotlistEventHandler {
 	private String appId;
 
 	/** The publisher. */
-	@Autowired
+	@Mock
```

Comment:
Mock should not be used in the code other than test classes.

Reply:
changed into Autowired